### ISSUE TITLE
Screeps seems to go in circles doing nothing.

### ISSUE THREAD

**Issue by 89gsc** [sscholl/screeps#3]
I have noticed that I end up getting a group of screeps that simply copy one another and move in a circle doing nothing else:

http://prntscr.com/c0oi8r

That shows the 2 groups of 2 which seem to copy each others movements, there is also 4 arrows to the left which show the pattern of movement they have., 

Any ideas what is happening? 

They will eventually stop doing it or what happens is they break off into 2 groups doing 2 different patterns of movement.... its weird.

**Comment by 89gsc** [comment]
Just to confirm this seems to be happening less now but I still notice the odd screep going in a circle usually its matching another creep at the other side of the room moving in the same way its really odd.

**Comment by sscholl** [comment]
The towers must be filled up. Are they doing that?

**Comment by 89gsc** [comment]
Yeah they are but I notice this more in remote rooms than controlled rooms now, like if a construction ID is invalid or is a destination is unreachable you need to manually remove the associated task from the room and clear it from the screep for it to fix itself.
